packages/client/components/developers/DeveloperList.tsx

```diff
@@ -4,19 +4,12 @@ import DeveloperItem from './DeveloperItem';
 
 export default function DeveloperList() {
   return (
-    <Box
-      position="relative"
-      textAlign="center"
-      m="auto"
-      h="100%"
-    >
+    <Box position="relative" textAlign="center" m="auto" h="100%">
       <Grid
         templateColumns={{
           '2xl': 'repeat(4, 2fr)',
           xl: 'repeat(3, 2fr)',
-          lg: 'repeat(2, 2fr)',
           md: 'repeat(2, 2fr)',
-          sm: 'repeat(1, 1fr)',
           base: 'repeat(1, 1fr)',
```

Comment:
Since `base` and `sm` are identical and `lg` and `md` are identical, we don't need to define `sm` or `lg`. `sm` falls back to `base` and `lg` falls back to `md`. (Type 2)

Reply:
MAAAN, I've been doing this wrongly for so long